cmd/nodecmd/wiz.go

```diff
@@ -326,21 +338,29 @@ func waitForClusterSubnetStatus(
 }
 
 func checkClusterIsADevnet(clusterName string) error {
-	exists, err := clusterExists(clusterName)
+	network, err := getClusterNetwork(clusterName)
 	if err != nil {
 		return err
 	}
+	if network.Kind != models.Devnet {
+		return fmt.Errorf("cluster %q is not a Devnet", clusterName)
+	}
+	return nil
+}
+
+func getClusterNetwork(clusterName string) (models.Network, error) {
+	exists, err := clusterExists(clusterName)
+	if err != nil {
+		return models.UndefinedNetwork, err
+	}
 	if !exists {
-		return fmt.Errorf("cluster %q does not exists", clusterName)
+		return models.UndefinedNetwork, fmt.Errorf("cluster %q does not exists", clusterName)
 	}
 	clustersConfig, err := app.LoadClustersConfig()
 	if err != nil {
-		return err
-	}
-	if clustersConfig.Clusters[clusterName].Network.Kind != models.Devnet {
-		return fmt.Errorf("cluster %q is not a Devnet", clusterName)
+		return models.UndefinedNetwork, err
 	}
-	return nil
+	return clustersConfig.Clusters[clusterName].Network, nil
```

Comment:
check if clustersConfig.Clusters[clusterName] is not nil before accessing Network